### ISSUE TITLE
Exception in preload leads to flash of content

### ISSUE THREAD

**Issue by IlyaSemenov** [sveltejs/sapper#710]
### Steps to reproduce

```
npx degit sveltejs/sapper-template#webpack my-app
```

create `src/routes/test.svelte`:

```html
<script context="module">
export async function preload(page, session) {
	await this.fetch('http://nosuchhost')
}
</script>

<script>
export let user
</script>

<div>user = {user}</div>
```

### Actual result

Server-side generated HTML contains error message:

```html
<h1 class="svelte-8od9u6">500</h1>

	<p class="svelte-8od9u6">request to http://nosuchhost/ failed, reason: getaddrinfo ENOTFOUND nosuchhost nosuchhost:80</p>

	<pre>FetchError: request to http://nosuchhost/ failed, reason: getaddrinfo ENOTFOUND nosuchhost nosuchhost:80
    at ClientRequest.eval (webpack:///./src/node_modules/@sapper/server.mjs?:1955:11)
    at ClientRequest.emit (events.js:182:13)
    at Socket.socketErrorListener (_http_client.js:391:9)
    at Socket.emit (events.js:182:13)
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at process._tickCallback (internal/process/next_tick.js:63:19)</pre>
```

It is displayed for a moment in the browser, but then replaced with `user = undefined`.

In javascript console there is a message: `<Test> was created without expected prop 'user'`

### Expected result

The error message 500 stays there. No errors in javascript console.

**Comment by nicobrinkkemper** [comment]
I'm still having this issue with Sapper

**Comment by bzin** [comment]
I just bump into this issue while fetching data from a GraphQL api endpoint. 

It only occurs when i open the route the first time, refreshing the page and  data is shown just fine. 

```
<script context="module">
    export async function preload (page, session) {
        // Fetch GraphQL query with homepage data
        const queryHomepage = await require('./../queries/homepage.graphql');

        // Url from GraphQL endpoint
        const url = process.env.GRAPHQL_API
        const opts = {
            method: 'POST',
            headers: { 'Content-Type': 'application/graphql' },
            body: queryHomepage
        };
        
        try {
            const res = await this.fetch(url, opts);
            const data = await res.json();
            return { post: data };
        } catch(error) {
            console.log(error)
        }
    }
</script>
```

**Comment by Banjerr** [comment]
I believe that is also what I am experiencing; wasn't sure if it was my implementation though.

https://github.com/Banjerr/Grateful-ness

**Comment by softmantk** [comment]
does anybody found a solution for this

**Comment by Conduitry** [comment]
Should be fixed in 0.27.14.
